File: Day 020/70. Climbing Stairs.cpp
class Solution {
public:
    int f(int n){
        int a =1;
        int b=2;
        int x;
        for (int i =2;i<n;i++){
            x = a +b;
            a=b;
            b=x;
        }
        return x;
    }
    int climbStairs(int n) {
        if (n==1){
            return 1;
        }
        else if (n==2){
            return 2;
        }
        else{
            return f(n);
        }
    }
};

File: Day 020/70. Climbing Stairs.py
class Solution(object):
    def climbStairs(self, n):
        a = 1
        b = 0
        for i in range(1, n+1):
            x = a + b
            b = a
            a = x
        return a
        """
        :type n: int
        :rtype: int
        """
